### PR TITLE
Update checkers.jsp

### PR DESCRIPTION
**Pull request by chunterme**
added code

### CODE REVIEW

File: CheckersClient/src/main/webapp/checkers.jsp [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+    text-align: center;
+    color: white;
+    font-family: verdana;
+    font-size: 20px;
+}
+</style>
+<body style="background-color:red;">
+<div>
+<br><h3>Welcome to the Checkers Game.</h3><br>
+<br><h3><a href = https://simple.wikipedia.org/wiki/Checkers>How to Play!</a> </h3><br>
```
Comment (LadyMozzarella) [inline CheckersClient/src/main/webapp/checkers.jsp:20]: Does this link work without quotes?

```html
<a href="https://simple.wikipedia.org/wiki/Checkers">How to Play!</a>
```

Comment (chunterme) [reply CheckersClient/src/main/webapp/checkers.jsp:20]: Is it not working? When I tested it, it was working?

checked by source:
Example
The href attribute specifies the link's destination:

<a href="http://www.w3schools.com">Visit W3Schools</a>

Comment (chunterme) [reply CheckersClient/src/main/webapp/checkers.jsp:20]: http://www.w3schools.com/tags/att_a_href.asp

**Comment by LadyMozzarella** [comment]
Goes with #4

File: CheckersClient/src/main/webapp/checkers.jsp (hunk 1 above)
Comment (chunterme) [reply CheckersClient/src/main/webapp/checkers.jsp:20]: fixed I didn't see that sorry.

Comment (LadyMozzarella) [reply CheckersClient/src/main/webapp/checkers.jsp:20]: No problem! I hadn't tested it. I just wasn't sure if that would work that way. 👍

**Pull request merged**